.changes/v0.12.7.md

```diff
@@ -0,0 +1,12 @@
+## v0.12.7 - 2024-09-02
+
+
+### Fixed
+- Fixed regression in module ref parsing for no-protocol refs that had version suffixes by @jedevc in https://github.com/dagger/dagger/pull/8298
+- Make `Git.tags` URL parsing consistent with `Git.tree` by @jedevc in https://github.com/dagger/dagger/pull/8298
```

Comment:
This reads as 2 fixed, when there's just one. Want to merge & maybe reword @jedevc?

Reply:
OK!